.github/workflows/check.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: Check 🛠
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  audit:
+    name: Audit Dependencies 🕵️‍♂️
+    uses: insightsengineering/r.pkg.template/.github/workflows/audit.yaml@main
```

Comment:
What do you mean?

Reply:
Oh yea, because that is where the original workflow resides: https://github.com/insightsengineering/r.pkg.template/blob/main/.github/workflows/audit.yaml